requirements.txt

```diff
@@ -1,13 +1,28 @@
+aiogram==2.3
+aiohttp==3.5.4
 appdirs==1.4.3
 astroid==2.2.5
+async-timeout==3.0.1
 attrs==19.1.0
+Babel==2.7.0
 black==19.3b0
+certifi==2019.6.16
+chardet==3.0.4
 Click==7.0
+dnspython==1.16.0
+idna==2.8
 isort==4.3.21
 lazy-object-proxy==1.4.1
+marshmallow==2.20.1
 mccabe==0.6.1
+multidict==4.5.2
 pylint==2.3.1
+pymongo==3.9.0
```

Comment:
You installed umongo with the pymongo driver. But, as we will write the async code for Telegram, we need the async database connection. So, please, remove the pymongo and add the Motor driver. The umongo with motor can be installed with `pip install umongo[motor]`. Check out [this](https://github.com/Scille/umongo/blob/master/README.rst)